crates/sdk/src/lib.rs

```diff
@@ -1123,7 +1123,7 @@ pub mod testing {
     prop_compose! {
         /// Generate an arbitrary masp transfer transaction
         pub fn arb_transfer_tx()(
-            mut header in arb_header(),
+            mut header in arb_header(0..=0),
```

Comment:
nit
```suggestion
            mut header in arb_header(0),
```
it turns into `0..=0`

Reply:
Thanks!